docker-compose.dev.yml

```diff
@@ -1,10 +1,28 @@
 version: "3.7"
 services:
+
+  vespatest:
+    image: vespaengine/vespa:8.273.20
+    container_name: vespatest
+    ports:
+      - 8080:8080
+      - 19071:19071
+    healthcheck:
+      test: curl -f http://localhost:19071/status.html || exit 1
```

Comment:
Is there a reason we're referring to this via localhost rather than the intended container name (like we do in the `docker-compose.yml` file)?

Reply:
Typically, we've been discarding the stdout of the healthcheck e.g., `curl -s -f http://localhost:19071/status.html >/dev/null || exit 1`. Not sure whether that might be doing here?

Reply:
Good spot, Updating this now